pylib/openrazer/client/fx.py

```diff
@@ -68,6 +68,46 @@ def __init__(self, serial: str, capabilities: dict, daemon_dbus=None, matrix_dim
 
         self.misc = MiscLighting(serial, capabilities, self._dbus)
 
+    @property
+    def effect(self) -> str:
+        """
+        Get current effect
+
+        :return: Effect name
+        :rtype: str
+        """
+        return self._lighting_dbus.getEffect()
+
+    @property
+    def colors(self) -> bytearray:
+        """
+        Get current effect colors
+
+        :return: Effect colors
+        :rtype: bytearray
+        """
```

Comment:
Please add example values to the doc

Reply:
`grep -ri example` under pylib/ returns nothing. What is this supposed to be?

Reply:
Replied in https://github.com/openrazer/openrazer/pull/888#discussion_r365511694

Reply:
Done